src/Altinn.App.Core/Helpers/ObjectUtils.cs

```diff
@@ -155,6 +173,11 @@ value is string s
         }
     }
 
+    [GeneratedRegex(
+        @"(?<![\uD800-\uDBFF])[\uDC00-\uDFFF]|[\uD800-\uDBFF](?![\uDC00-\uDFFF])|[\x00-\x08\x0B\x0C\x0E-\x1F\x7F-\x9F\uFEFF\uFFFE\uFFFF]"
+    )]
```

Comment:
Can you explain this regex?

Reply:
Added comment referring to the origin. I think regexr is a better tool for explanation of the actual regex: https://regexr.com/870t1